src/python/pants/reporting/workunits.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import json
+from typing import Tuple
+
+import requests
+from requests.exceptions import RequestException
+
+from pants.subsystem.subsystem import Subsystem
+
+
+class Workunits(Subsystem):
+  options_scope = 'workunits'
+
+  @classmethod
+  def register_options(cls, register):
+    register("--http-endpoint", type=str, help="Where to make HTTP requests to")
```

Comment:
This feels pretty related to the RunTracker. Although it already has a lot of options, it would be very good to try to define these there (and to think about what they might deprecate) rather than creating a new Subsystem that is highly overlapped.